core/iwasm/libraries/lib-pthread/lib_pthread_wrapper.c

```diff
@@ -532,7 +532,7 @@ pthread_create_wrapper(wasm_exec_env_t exec_env,
     bh_assert(module_inst);
 
     if (!(new_module_inst = wasm_runtime_instantiate_internal(
-              module, true, 8192, 0, NULL, 0)))
+              module, true, APP_THREAD_STACK_SIZE_DEFAULT, 0, NULL, 0)))
```

Comment:
Use the original module_inst's  operand stack size should be more reasonable: module_inst->default_wasm_stack_size, which is the stack size used to instantiate the original module inst.
APP_THREAD_STACK_SIZE_DEFAULT is the default native stack size.